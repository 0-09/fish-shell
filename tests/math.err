
####################
# Validate basic expressions

####################
# Validate how bare variables in an epxression are handled
