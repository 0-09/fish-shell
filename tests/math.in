math 3 / 2
math 10/6
math -s0 10 / 6
math -s3 10/6
math '10 % 6'
math -s0 '10 % 6'
math '23 % 7'
math --scale=6 '5 / 3 * 0.3'
math "7^2"
math -1 + 1
math '-2 * -2'
math 5 \* -2
math -- -4 / 2
math -- '-4 * 2'
