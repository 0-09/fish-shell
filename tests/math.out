1
1
1
1.667
4
4
2
0.500000
49
0
4
-10
-2
-8
